sdk/search/Azure.Search.Documents/tests/Utilities/SearchRecordedTestSanitizer.cs

```diff
@@ -96,7 +96,11 @@ private static string SanitizeVariable(ISet<string> secrets, string name, string
             if (SearchTestEnvironment.StorageAccountKeyVariableName.Equals(name, StringComparison.OrdinalIgnoreCase) ||
                 SearchTestEnvironment.ClientSecretVariableName.Equals(name, StringComparison.OrdinalIgnoreCase))
             {
-                // Assumes the secret content is destined to appear in JSON, for which certain common characters in account keys are escaped.
+                // The secret content could be embedded within a string. Like -
+                // "DefaultEndpointsProtocol=https;AccountName={StorageAccountName};AccountKey={StorageAccountKey};EndpointSuffix={StorageEndpointSuffix}"
```

Comment:
Make sure this doesn't trip cred scan?  It can be kind of dumb.

Reply:
I'll remove it to get annoyed down the line.